### PR TITLE
Feature/add grpc

### PR DESCRIPTION
**Pull request by Sandro-Meireles**
Proto file PR: Ilhasoft/rapidpro-apps-protobuffers#1

### CODE REVIEW

File: weni/org_grpc/serializers.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    def _user_has_permisson(self, user: User, org: Org) -> bool:
+        return user.org_admins.filter(pk=org.pk) \
+            or user.org_viewers.filter(pk=org.pk) \
+            or user.org_editors.filter(pk=org.pk) \
+            or user.org_surveyors.filter(pk=org.pk)
+
+    class Meta:
+        model = Org
+        proto_class = org_pb2.Org
+        fields = ["id", "user_id", "name", "timezone", "date_format", "plan",
```
Comment (matmsa27) [inline weni/org_grpc/serializers.py:115]: ran `black` here?

File: weni/org_grpc/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from weni.org_grpc.grpc_gen import org_pb2_grpc
+
+from weni.org_grpc.services import OrgService
+
+
+def grpc_handlers(server):
+    org_pb2_grpc.add_OrgControllerServicer_to_server(OrgService.as_servicer(), server)
```
Comment (matmsa27) [inline weni/org_grpc/urls.py:7]: please, add the newline at end of the file

**Review by matmsa27 — CHANGES_REQUESTED** [review]
Great work Sandro, congrats, I believe that is missing the `black` on the code to follow the Rapidpro conventions. Thanks 💯

File: weni/org_grpc/serializers.py (hunk 1 above)
Comment (Sandro-Meireles) [reply weni/org_grpc/serializers.py]: black was runned and I pushed a new commit: https://github.com/Ilhasoft/rapidpro-apps/pull/11/commits/b55eab767899e1ad7db8ead2b91eda78cc05a361

File: weni/org_grpc/urls.py (hunk 2 above)
Comment (Sandro-Meireles) [reply weni/org_grpc/urls.py]: I made the change and pushed on a new commit: https://github.com/Ilhasoft/rapidpro-apps/commit/b55eab767899e1ad7db8ead2b91eda78cc05a361

**Review by matmsa27 — APPROVED** [review]

**Review by jcbalmeida — COMMENTED** [review]
It would be nice to have a README with instructions on how to configure the app and run the server.
The required libraries can be added as requirements in setup.py too.

**Review by jcbalmeida — APPROVED** [review]

**Comment by jcbalmeida** [comment]
We will make some improvements in the next PR, good job!

**Pull request merged**